### PR TITLE
JSON-LD option added

### PR DESCRIPTION
**Pull request by Nikhil-lab**
- CsvUrl was created and jsonld is passed as output parameter to the v2/csv endpoint.

### CODE REVIEW

File: projects/v2/src/app/modules/root/root.component.ts [hunk 1]
```diff
@@ -398,6 +422,24 @@ export class RootComponent implements OnInit, OnDestroy {
       document.body.appendChild(downloadAnchorNode);
       downloadAnchorNode.click();
       downloadAnchorNode.remove();
+    } else if (option === 'JSON LD') {
```
Comment (bherr2) [inline projects/v2/src/app/modules/root/root.component.ts:425]: Option should be named "JSON-LD"

File: projects/v2/src/app/services/sheet.service.ts [hunk 2]
```diff
@@ -6,29 +6,44 @@ import { map } from 'rxjs/operators';
 import { SheetInfo, Structure } from '../models/sheet.model';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
```
Comment (bherr2) [inline projects/v2/src/app/services/sheet.service.ts:9]: no trailing commas

File: projects/v2/src/app/services/sheet.service.ts [hunk 3]
```diff
@@ -107,19 +124,19 @@ export class SheetService {
...
    * Service to add body for each AS to the data
    * @param data is the parsed ASCTB data from the csv file of the sheet
    */
-  getDataWithBody(data: any) {
+  getDataWithBody(data: any, organName: string) {
     const organ: Structure = {
       name: 'Body',
       id: 'UBERON:0013702',
-      rdfs_label: 'body proper'
+      rdfs_label: 'body proper',
```
Comment (bherr2) [inline projects/v2/src/app/services/sheet.service.ts:135]: no trailing commas

File: projects/v2/src/app/static/config.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { SheetDetails } from '../models/sheet.model';
 
 export const HEADER_COUNT = 11;
 
-export const IMG_OPTIONS = ['PNG', 'SVG', 'Vega Spec', 'Graph Data'];
+export const IMG_OPTIONS = ['PNG', 'SVG', 'Vega Spec', 'Graph Data', 'JSON LD'];
```
Comment (bherr2) [inline projects/v2/src/app/static/config.ts:5]: JSON-LD

Comment (Nikhil-lab) [reply projects/v2/src/app/static/config.ts:5]: ok Bruce. I will make those changes now.

**Comment by bherr2** [comment]
This still has trailing commas that need to be fixd. Please look closely at https://github.com/hubmapconsortium/ccf-asct-reporter/pull/233/files

**Comment by bherr2** [comment]
Please pull from `develop` and fix merge conflicts. Check out how Ajay updated the Graph Data export to see how to support multiple selected organs and exporting.

**Comment by bherr2** [comment]
Why am I seeing 34 files changed!?

**Comment by vutukuriajay2241** [comment]
> Why am I seeing 34 files changed!?

Instead of pulling code from develop branch. @Nikhil-lab  has rebased his branch with Develop. So it's possible that instead of showing "Merge from" its showing all the commits behind that.

**Comment by Nikhil-lab** [comment]
> Why am I seeing 34 files changed!?

I have not done changes in all 34 files but I had just resolved conflicts during git rebase and made a new commit.

**Review by bherr2 — APPROVED** [review]

**Comment by bherr2** [comment]
Ok.. Try not to do that again. It makes it very hard to review.

**Comment by Nikhil-lab** [comment]
> Ok.. Try not to do that again. It makes it very hard to review.

Sure Bruce

**Pull request merged**